ledger/error/src/main/scala/com/daml/error/BaseError.scala

```diff
@@ -101,15 +100,6 @@ object BaseError {
     msg.startsWith(SecuritySensitiveMessageOnApiPrefix)
   }
 
-  def extractContext[D](obj: D): Map[String, String] =
-    obj.getClass.getDeclaredFields
-      .filterNot(x => ignoreFields.contains(x.getName) || x.getName.startsWith("_"))
-      .map { field =>
-        field.setAccessible(true)
-        (field.getName, field.get(obj).toString)
-      }
-      .toMap
-
```

Comment:
@pbatko-da @mziolekda Did you check with Canton wrt to this change? I've discussed with Ratko a couple of months ago re this, but he did not want to drop the reflective gathering of context.

Reply:
Great. Thanks @rgugliel-da